openreview/venue_request/process/postSubmissionProcess.py

```diff
@@ -1,6 +1,31 @@
 def process(client, note, invitation):
-
+    GROUP_PREFIX = ''
```

Comment:
are you setting this variable?

Reply:
It's the same as in deployProcess and revisionProcess.

Reply:
right, you need to set the value in the python code. Like we are doing here:

https://github.com/openreview/openreview-py/blob/65bfa35d2a242674abd4b7cf3a8d9f3762b12a11/openreview/venue_request/venue_request.py#L10

You added this variable to the postSubmissionProcess.py but you are not setting any value:

https://github.com/openreview/openreview-py/blob/65bfa35d2a242674abd4b7cf3a8d9f3762b12a11/openreview/venue_request/venue_request.py#L984

Reply:
Thanks, this worked.